### ISSUE TITLE
Bug report

### ISSUE THREAD

**Issue by sanqusanhui** [danielsreichenbach/mod-playerbots#1]
Thank you for your repair. I found that BG robots are only in a daze at the place of birth, and will not follow the instructions in BattleGroundTactics to seize flags, align lines, etc. Can u fix that?

**Comment by danielsreichenbach** [comment]
Thank you for the report. I will investigate soon, still checking out how to update the module with all the changes in the original IKE3 bot project.

**Comment by knuxyl** [comment]
Yes after 24 hours bots are standing around and dancing in starter area. 1 got to level 8, but was still around. Bots are not progressing.
